libqaul/Cargo.toml

```diff
@@ -1,13 +1,14 @@
 [package]
 name = "libqaul"
 version = "0.1.0"
-authors = ["Katharina Fey <kookie@spacekookie.de>"]
+authors = ["Katharina Fey <kookie@spacekookie.de>",
+		   "Jess 3Jane <me@jess.coffee>"]
 edition = "2018"
 
 [dependencies]
+iron = "0.6"
```

Comment:
This looks like it should only be a dependency of `http-api` ?

Reply:
presently its a dependency here because error implements `From<HttpError>` and `From<Error> for IronError`